tools/requirements.txt

```diff
@@ -0,0 +1,16 @@
+setuptools==50.3.2
```

Comment:
Shouldn't these requirements not be bound to a version?

Reply:
Yeah these need not be version bounded, removed them in new revision.